### ISSUE TITLE
Exclude Mediaqueries didn't catch.

### ISSUE THREAD

**Issue by marcus-at-localhost** [tijsverkoyen/CssToInlineStyles#32]
Hello, 
I was wondering why my Mediaqueries weren't excluded from inlining.
it was something like this:

``` css
@media only screen and (max-width: 480px){ }
```

and

``` php
$css = preg_replace('/@media [^{]*{([^{}]|{[^{}]*})*}/', '', $css);
```

didn't catch.

So I found another regex snippet that worked, in [responsiv.js](https://github.com/scottjehl/Respond/blob/master/respond.src.js)

so this works now:

``` php
$css = preg_replace('/@media[^\{]+\{([^\{\}]*\{[^\}\{]*\})+/', '', $css);
```

I didn't break down the regex because I needed a quick fix.

Btw. I used the MailChimp Responsive E-Mail Templates in this case.
